Editors/TiledEditor/source/TiledLevel.hx

```diff
@@ -91,21 +96,68 @@ class TiledLevel extends TiledMap
 		}
 	}
 	
-	public function loadObjects(state:PlayState)
+	public function loadObjects()
```

Comment:
Why remove the `state` parameter here? Seems nicer to pass it than to create a static reference to it that's only needed in a single function.